### PR TITLE
UI feature auto complete

### PR DESCRIPTION
**Pull request by iguannalin**
## Description

Updated/Added auto-complete feature to agents, adversaries and abilities page to search for abilities 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: templates/profiles.html [hunk 1]
```diff
@@ -299,9 +299,9 @@ <h3>Create a profile</h3>
                         </div>
                         <div class="field-body">
                             <div class="field">
-                                <div class="control">
+                                <div class="control auto-complete">
```
Comment (argaudreau) [inline templates/profiles.html:302]: The autocomplete on the search works great here! I'd add an autocomplete to the tactic, technique Id and name just like I suggested for the abilities page

**Review by argaudreau — CHANGES_REQUESTED** [review]
Great idea moving the dropdown styles to shared.css so it's really easy to add if we need it anywhere else in the future. Just a couple of tweaks to make but overall looks good!

File: templates/abilities.html [hunk 2]
```diff
@@ -167,14 +167,38 @@ <h2>Abilities</h2>
...
                             </div>
+                            <div class="field is-horizontal">
+                                <div class="field-label is-small">
+                                    <label class="label">
+                                        <span class="icon is-small"><em class="fas fa-search"></em></span>
+                                    </label>
+                                </div>
+                                <div class="field-body">
+                                    <div class="field">
+                                        <div class="control auto-complete" x-data="{focusSearchResults: true}">
```
Comment (argaudreau) [inline templates/abilities.html:178]: We can remove the search from this modal since there's a search on the page for it.

**Review by argaudreau — CHANGES_REQUESTED** [review]
Just a couple more things and we'll be good to merge!

File: templates/abilities.html [hunk 3]
```diff
@@ -167,14 +167,38 @@ <h2>Abilities</h2>
...
                             <div class="field is-horizontal">
                                 <div class="field-label is-small">
                                     <label class="label">Tactic</label>
                                 </div>
                                 <div class="field-body">
                                     <div class="field">
-                                        <div class="control">
-                                            <input class="input is-small" x-bind:class="{ 'is-danger': fieldErrors.includes('tactic') }" x-model="selectedAbility.tactic">
+                                        <div class="control auto-complete" x-data="{focusSearchResults: true}">
+                                            <input class="input is-small" x-bind:class="{ 'is-danger': fieldErrors.includes('tactic') }" x-model="selectedAbility.tactic" x-on:keyup="searchForAutoFill('tactic')" @click="focusSearchResults = true">
```
[the remark this answers is not in the document]
Comment (argaudreau) [reply templates/abilities.html:196]: Scratch that -- I'm no longer seeing this behavior

**Review by argaudreau — APPROVED** [review]

**Review by mshkolnik22 — APPROVED** [review]
Looks good, auto complete feature works.

**Pull request merged**